src/pages/program/index.astro

```diff
@@ -0,0 +1,70 @@
+---
+import Layout from "../../layouts/Layout.astro";
+import type { Session } from "types/program";
+import { fetchProgram } from "./fetchProgram";
+import SessionCard from "../../components/SessionCard.astro"
+
+const fullprogram = await fetchProgram()
+
+const porgramReady = false;
+
+// function filterSessionsByLength(sessions: Session[]): Session[] {
```

Comment:
Trengs all denne kommenterte koden?

Reply:
Mye av koden er egentlig en mer oversiktlig program-side, men har kommentert det ut nå for den midlertidige oversikten som bare er en liste. Så det er det jeg bruker videre til neste versjon av programsiden. Er det ok å la det stå, eller skal jeg flyttet det ut og kun ha det lokalt for nå?

Reply:
View til Individuell talk funker hos meg, usikker på hvorfor den ikke henter ut noe hos deg. Gjelder det alle talks?

Reply:
Det funker ikke når jeg prøver preview urlen: https://vercel.live/open-feedback/2024-javazone-no-git-program-javabin.vercel.app?via=pr-comment-visit-preview-link&passThrough=1

Reply:
I local på f.eks http://localhost:4321/program/0748c9b0-f779-47af-ae5b-ef3fa678de00
funker det fint. Får du riktig slug på slutten når du klikker deg inn?

Reply:
Hmm, ja, tittet der nå. Skal se nærmere på det. Den kommer til rett view, men får tydeligvis ingen session av en eller annen grunn.

Reply:
Hvordan ser det ut hos deg nå etter siste endring?

Reply:
Får opp sessions nå!